config/log.conf

```diff
@@ -7,8 +7,10 @@ log_format main_json escape=json
   '{'
     '"body_bytes_sent":"$body_bytes_sent",'
     '"host":"$host",'
-    '"http_referrer":"$http_referer",'
+    '"http_connection":"$http_connection",'
+    '"http_referer":"$http_referer",'
     '"http_user_agent":"$http_user_agent",'
+    '"http_x_amzn_trace_id":"$http_x_amzn_trace_id",'
```

Comment:
This way, one can correlate the `X-Request-Id` with the `X-Amzn-Trace-Id` (if it exists).